edx_django_utils/cache/tests/test_middleware.py

```diff
@@ -2,8 +2,6 @@
 """
 Tests for the RequestCacheMiddleware.
 """
-# pylint: disable=missing-docstring
```

Comment:
Maybe this is just personal preferences, but I added the disable pylint to avoid docstrings that basically just repeat the names of the classes.  So, I personally prefer not making these changes.

Reply:
Ok, I reverted those redundant docstrings. I had thought that pylint was complaining about it, somehow.